release-please-config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "bootstrap-sha": "0d71fd1508911c10b9944f661c9e109a7dc1e23c",
```

Comment:
Is this commit legit?  Did a quick check on github and didn't get a hit in this repo.